eks-anywhere-common/Addons/Partner/Lacework/Polygraph/cluster-secret-store.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
Please remove this file. Our framework creates `ClusterSecretStore` this to sync secrets in AWS Secrets Manager to the cluster so this is not required.